doc/source/security/encrypting-undo-tablespace.rst

```diff
@@ -0,0 +1,43 @@
+.. _undo-tablespace-encryption:
+
+================================================================================
+Encrypting the Undo Tablespace
+================================================================================
+
+The undo data may contain sensitive information about the database operations.
+You can encrypt the data in an undo log using the :variable:
+`innodb_undo_log_encrypt` option. You can change the setting for this variable
+in the configuration file. The undo data encryption must be enabled; the feature
+is disabled by default.
+
+.. variable:: innodb_undo_log_encrypt
+
+    :cli: ``--innodb_undo-log_encrypt``
+    :dyn: Yes
+    :scope: Global
+    :vartype: Boolean
+    :default: OFF
+
+Defines if an undo log data is encrypted in separate undo tablespaces. The default for the undo log is "OFF", which disables the encryption.
```

Comment:
MySQL 8.0 only has separate undo tablespaces, there is no other option.

Reply:
fixed.